clisops/ops/subset.py

```diff
@@ -95,9 +134,11 @@ def subset(
                 Union[int, float, str],
                 Union[int, float, str],
             ],
+            AreaParameter
         ]
     ] = None,
-    level: Optional[Union[str, Tuple[Union[int, float, str], Union[int, float, str]]]] = None
+    level: Optional[
+        Union[str, Tuple[Union[int, float, str], Union[int, float, str]], LevelParameter]
```

Comment:
```suggestion
    level: Optional[Union[str, Tuple[Union[int, float, str], Union[int, float, str]], LevelParameter]
```

Reply:
This needs to stay on one line, otherwise it will be treated as flavour text for the call signature.